custom_labelbox.py

```diff
@@ -79,6 +81,7 @@ def __init__(
         members=None,
         classes_as_attrs=True,
         upload_media=True,
+        iam_integration_name=None,
```

Comment:
Would this need to be `"DEFAULT"`?

Reply:
Ah, yes. I've updated that in a few places in 94a94f05db067429e048a2cb2ccb199437490a50.